apps/src/code-studio/components/progress/StudentTable.jsx

```diff
@@ -61,7 +80,11 @@ class StudentTable extends React.Component {
             style={this.getRowStyle(selectedUserId, null)}
             onClick={() => onSelectUser(null)}
           >
-            <td style={styles.td}>{i18n.studentTableTeacherDemo()}</td>
+            <td style={styles.td}>
```

Comment:
instead of having nested elements (`td > span`) to add the `styles.nameInScript` styles, you should be able to use the spread operator to apply both styles to the `<td/>` element:

```
<td style={{...styles.td, ...styles.nameInScript}}>
  {i18n.studentTableTeacherDemo()}
</td>
```